libr/anal/fcn.c

```diff
@@ -414,11 +415,47 @@ static bool is_delta_pointer_table (RAnal *anal, ut64 ptr) {
 	return true;
 }
 
+static bool regs_exist(RAnalValue *src, RAnalValue *dst)
+{
+	if (src && dst && src->reg && dst->reg && src->reg->name && dst->reg->name) {
```

Comment:
yeah, better. There was some logic before, and after chage I forgot to remove this conditions.